### PR TITLE
FS-3535/FS-4340: implementation of the SQS extended client for the assessment store

### PR DESCRIPTION
**Pull request by nuwan-samarasinghe**
https://dluhcdigital.atlassian.net/browse/FS-3535
https://dluhcdigital.atlassian.net/browse/FS-4340

### Change description
assessment store sqs extended client changes to send messages to notification service queue

- [x] Unit tests and other appropriate tests added or updated
- [ ] README and other documentation has been updated / added (if needed)
- [x] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
This should successfully import the completed applications which is in the SQS queue

### CODE REVIEW

File: _helpers/context_aware_executor.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+from concurrent.futures import ThreadPoolExecutor
+from contextvars import copy_context
+
+
+class ContextAwareExecutor:
```
Comment (srh-sloan) [inline _helpers/context_aware_executor.py:5]: Do we need to redefine this here as it's already in notification?
app/notification/scheduler/context_aware_executor.py
https://github.com/communitiesuk/funding-service-design-notification/pull/233/files#diff-f0964165f72c7f2ce4fd79324539d9c286a02df206da7f35498ad1fbf2433f96R5

File: _helpers/task_executer_service.py [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+from concurrent.futures import as_completed
+from os import getenv
+
+from config import Config
+from config.mappings.assessment_mapping_fund_round import fund_round_data_key_mappings
+from db.queries import bulk_insert_application_record
+from fsd_utils.services.aws_extended_client import SQSExtendedClient
+
+
+class TaskExecutorService:
```
Comment (srh-sloan) [inline _helpers/task_executer_service.py:12]: Does all of this need redefining, as a lot of it is common with notify? The receive and delete portions should do the same thing, but we could provide a callback to do the processing?

**Review by srh-sloan — CHANGES_REQUESTED** [review]
Couple of comments around whether we can reuse more stuff that's in notify. Happy to discuss.

File: _helpers/context_aware_executor.py (hunk 1 above)
Comment (nuwan-samarasinghe) [reply _helpers/context_aware_executor.py]: Fixed

File: _helpers/task_executer_service.py (hunk 2 above)
Comment (nuwan-samarasinghe) [reply _helpers/task_executer_service.py]: Fixed

**Pull request merged**